### PR TITLE
Adding cypress test for listing location relative criteria

### PR DESCRIPTION
**Pull request by iFlameing**
Related to https://github.com/plone/volto/pull/1862

### CODE REVIEW

File: cypress/integration/blocks-listing.js [hunk 1]
```diff
@@ -425,7 +395,87 @@ if (Cypress.env('API') !== 'guillotina') {
         .contains('Document outside Folder')
         .should('not.exist');
     });
-    it('Listing block - Test Criteria: Location Navigation path', () => {
+    it('Listing block - Test Criteria: Location relative particular to a issue', () => {
+      cy.createContent({
```
Comment (tisto) [inline cypress/integration/blocks-listing.js:399]: @iFlameing could you please add the lines from my BDD scenario (GIVEN, WHEN, THEN) as subheadlines to the test for better readability?

**Comment by iFlameing** [comment]
@tisto I will update it today :)

File: cypress/integration/blocks-listing.js (hunk 1 above)
Comment (iFlameing) [reply cypress/integration/blocks-listing.js:399]: onto it

Comment (iFlameing) [reply cypress/integration/blocks-listing.js:407]: Done :)

**Review by sneridagh — APPROVED** [review]

**Pull request merged**